### PR TITLE
kubeadm,kubelet: Add versions 1.16.3

### PR DESCRIPTION
**Pull request by benmoss**
Let me know if you have any questions!

### CODE REVIEW

File: bucket/kubeadm.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://dl.k8s.io/v1.16.3/bin/windows/amd64/kubeadm.exe",
+            "hash": "sha1:4d953f7245544ceb86101e1e6c34342c3e0fb230"
+        }
+    },
+    "bin": "kubeadm.exe",
+    "checkver": {
+        "url": "https://storage.googleapis.com/kubernetes-release/release/stable.txt",
+        "re": "v([\\d.]+)"
```
Comment (Ash258) [inline bucket/kubeadm.json]:
```suggestion
        "regex": "v([\\d.]+)"
```

File: bucket/kubeadm.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/",
+    "description": "Kubernetes node provisioning tool.",
```
Comment (Ash258) [inline bucket/kubeadm.json]:
```suggestion
    "description": "Kubernetes node provisioning tool",
```

File: bucket/kubelet.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://dl.k8s.io/v1.16.3/bin/windows/amd64/kubelet.exe",
+            "hash": "sha1:f788a8d53802beb18f3c150020a4994bfebb3734"
+        }
+    },
+    "bin": "kubelet.exe",
+    "checkver": {
+        "url": "https://storage.googleapis.com/kubernetes-release/release/stable.txt",
+        "re": "v([\\d.]+)"
```
Comment (Ash258) [inline bucket/kubelet.json]:
```suggestion
        "regex": "v([\\d.]+)"
```

File: bucket/kubelet.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/",
+    "description": "Kubernetes node agent.",
```
Comment (Ash258) [inline bucket/kubelet.json]:
```suggestion
    "description": "Kubernetes node agent",
```

File: bucket/kubeadm.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    "license": "Apache-2.0",
```
Comment (Ash258) [inline bucket/kubeadm.json]: Where did you get this information?

Comment (benmoss) [reply bucket/kubeadm.json]: https://github.com/kubernetes/kubernetes/blob/master/LICENSE
kubeadm is an application that is part of the kubernetes/kubernetes repo: https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm

**Comment by benmoss** [comment]
Oops, sorry for the re-review request, thought you might not see that I had pushed another commit

**Pull request merged**